vocabularies/borehole-status-event.ttl

```diff
@@ -11,19 +11,20 @@
     dcterms:creator <http://linked.data.gov.au/org/gsq> ;
     dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
     dcterms:created "2020-01-21"^^xsd:date ;
-    dcterms:modified "2020-06-17"^^xsd:date ;
+    dcterms:modified "2020-07-07"^^xsd:date ;
     dcterms:source "Compiled by the Geological Survey of Queensland" ;
     skos:definition "Events that occur during the lifecycle a borehole that affect the status or history, or convert a borehole from one purpose to another."@en ;
     skos:hasTopConcept bhse:drilling-of-borehole-on-notice,
+        bhse:drilling-of-borehole-abandoned,

```

Comment:
Should this be "Never drilled" as to clearly delineate it from bhse:abandonment?

Reply:
changed